tests/fast_tests/test_traffic_lights.py

```diff
@@ -194,6 +195,45 @@ def test_k_closest(self):
             self.assertTrue(self.env.vehicles.get_edge(veh_id) in c0_edges)
 
 
+    def test_min_switch(self):
+        # FOR THE PURPOSES OF THIS TEST, never set min switch to be < 2
```

Comment:
Does this test enforce the constraint of `min_switch >= 2`?

Reply:
The setup script it uses has min_switch >=2, is that what you mean?